examples/boston/cross_val.cfg

```diff
@@ -8,6 +8,7 @@ train_directory = train
 featuresets = [["example_boston_features"]]
 # there is only set of features to try with one feature file in it here.
 featureset_names = ["example_boston"]
+feature_scaling = both
```

Comment:
Please add a comment about why this is something worth doing - to put the various features on the same scale.